graphql_compiler/tests/test_compiler.py

```diff
@@ -776,7 +832,17 @@ def test_between_lowering_with_extra_filters(self):
                 name: m.Animal___1.name
             ])}
         '''
-        expected_sql = NotImplementedError
+        expected_sql = '''
+            SELECT
+                animal_1.name AS name
+            FROM
+                animal AS animal_1
+            WHERE
+                animal_1.name <= :upper
+                AND (animal_1.name LIKE '%' || :substring || '%')
+                AND animal_1.name IN ([EXPANDING_fauna])
```

Comment:
Yes that's the indication for SQLAlchemy to accept a list-valued input:
https://docs.sqlalchemy.org/en/latest/core/sqlelement.html#sqlalchemy.sql.expression.bindparam.params.expanding

Reply:
Interesting, did not know that. Might be good to add a link to that page in the part of code that causes this to be generated. Also, if possible, it might be good to (in a future PR) add a test that the generated SQL after parameters are injected looks as expected too.